demo/addons/godot-openxr/runtimes.json

```diff
@@ -0,0 +1,22 @@
+[
+    {
+        "name": "Monado",
+        "path": "/usr/share/openxr/1/openxr_monado.json"
+    },
+    {
+        "name": "SteamVR",
+        "path": "~/.steam/steam/steamapps/common/SteamVR/steamxr_linux64.json"
+    },
```

Comment:
@ChristophHaag I have no idea if the ~ works, we may need to add some code that it replaces it with the users home folder.

Reply:
It does not work, and kinf of expected, `$HOME` also isn't automatically resolved.

This would be the easy fix for linux
```
var home = OS.get_environment("HOME")
...
var path = runtime["path"].replace("~", home)
```

Reply:
same handling will be needed in _on_OpenXRRunSelect_item_selected

Reply:
Ok it's HOMEDRIVE and HOMEPATH on windows, so I'll add that in just for completeness sake.
I'm putting the logic to replace the home path into a function so we can extend it later on. Also keeping the path as is inside of the array so we can add items and save it back to a json at some point.